ocs_ci/deployment/deployment.py

```diff
@@ -1626,8 +1648,23 @@ def deploy_with_external_mode(self):
         # enable secure connection mode for in-transit encryption
         if config.ENV_DATA.get("in_transit_encryption"):
             external_cluster.enable_secure_connection_mode()
-
-        setup_ceph_toolbox()
+        if config.DEPLOYMENT.get("multi_storagecluster"):
+            logger.info("not setting toolbox in multi-storagecluster")
+            health_check_namespace = constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE
```

Comment:
Why we are not setting up toolbox pod for external cluster?
I think we should have two toolbox pods, one for internal mode - this will be created by default from non external cluster deployment. And we should have second toolbox pod - probably in this other namespace for external mode cluster - which will allow us to see status and health of external cluster.

Reply:
After I had problem in setting up toolbox in another namespace, Dev gave it a try too and after failure they asked me to work directly with the ceph cluster.

Reply:
I think the main problem is all rook resource are in another namespace and can't be shared like secret etc.